### DISCUSSION TITLE
Tweaking NPC Damage

### DISCUSSION THREAD

**Discussion by olandas84** [trickerer/Trinity-Bots#919] [General]
I've looked through the settings and apologize if it's already there, but it seems that the bots are hitting hard. I was hoping that there was a way to dial back their crit chances, or have a way to lessen there effectiveness as you higher more.

Sometimes I can barely engage with a mob before the bots have already taken it down.

**Comment by trickerer** [comment]
Everything you may need to tune bots' damage already exists in config.
- `NpcBot.Mult.Damage.<DMGTYPE>`
- `NpcBot.Mult.Damage.<BOTCLASS>`
- `NpcBot.Mult.Damage.Levels`
- `NpcBot.Stats.Limit.<COMBAT_STAT%>`

**Reply by xBarselonx** [reply]
А в настройке NpcBot.Mult.Damage.<Class> можно установить 1.5?

**Reply by trickerer** [reply]
> А в настройке NpcBot.Mult.Damage. можно установить 1.5?

Там же написано от 0.1 до 10.0
<details><summary>Details</summary>
<p>

Можно

</p>
</details>

**Reply by xBarselonx** [reply]
> > А в настройке NpcBot.Mult.Damage. можно установить 1.5?
> 
> Там же написано от 0.1 до 10.0
> 
> Details
> 
> Можно

Спасибо!!!

**Reply by xBarselonx** [reply]
А настройку NpcBot.Mult.Damage в игре командой можно менять?

**Reply by trickerer** [reply]
Нет. Так команд не напасёшься, если для каждого параметра добавлять. Есть команда для перезагрузки конфига ботов `.npcbot reloadconfig` / `.npcb rel`
